Runtime/Core/AmazonGameLiftPlugin.Core.csproj

```diff
@@ -20,6 +20,7 @@
     <PackageReference Include="Serilog.Sinks.File" Version="5.0.0" />
     <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="6.11.1" />
     <PackageReference Include="System.IO.Compression.ZipFile" Version="4.3.0" />
+    <PackageReference Include="Unity3D.SDK" Version="2021.1.14.1" />
```

Comment:
dumb question since i don't understand how this works! Is this supposed to be a Unity 2022.3 version, since that's what we're targeting finally?

Reply:
Or is that a separate change outside the scope of this PR?

Reply:
This is the most up to date version of the sdk. This is just so we have access to unity's features, this shouldn't cause any issue.